scripts/integration-test-all.sh

```diff
@@ -125,7 +125,7 @@ start_cli_func() {
     --laddr "tcp://localhost:$RPC_PORT$i" --node tcp://$IP_ADDR:$NODE_RPC_PORT"$i" \
     --home "$DATA_CLI_DIR$i" --read-timeout 30 --write-timeout 30 \
     >"$DATA_CLI_DIR"/cli"$i".log 2>&1 & disown
-    
+
     ETHERMINT_CLI_PID=$!
```

Comment:
i dont think we need to track ethermintcli pid anymore because we don't have ethermintcli anymore